lib/katello/plugin.rb

```diff
@@ -246,6 +246,10 @@
       :name => _("Content Sync"),
       :partial => "foreman/smart_proxies/content_sync",
       :onlyif => proc { |proxy| proxy.pulp_mirror? }
+    context.add_pagelet :reclaim_space,
```

Comment:
this should be :details_content

i think this is describing which 'section' to add the pagelet to